tests/cli/test_moe.py

```diff
@@ -19,9 +19,7 @@ def test_run_moe(click_runner, federated_ursulas):
 
     _ursula_output = yield threads.deferToThread(start_pytest_ursula_services, ursula=ursula)
 
-    test_ws_port = select_test_port()
```

Comment:
we can delete this entire file since we removed Moe - deleted myself in https://github.com/nucypher/nucypher/pull/963/commits/e3acb7788b21e10709775c0a813119694881b467